src/components/WishlistCard.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import { useCart } from '../context/CartContext';
+import { useWishlist } from '../context/WishContext';
+
+function WishlistCard({_id,src,categoryName,price,originalPrice,title,discount,rating}){
+    const {cartDispatch}=useCart();
+    const {wishDispatch}=useWishlist();
+
+    const moveToCart=()=>{
+        cartDispatch({
+            type:"MOVE_TO_CART",
+            moveItem:{
+                _id:_id,
+                categoryName:categoryName,
+                src:src,
+                title:title,
+                originalPrice:originalPrice,
+                price:price,
+                discount:discount,
+                rating:rating
```

Comment:
I think you can pass whole object of product here. so you have to no need to do like this

Reply:
yes i tried doing that way, but somehow screen was not rendering. I think i need to look on this.